graphql-cli/gradle.properties

```diff
@@ -0,0 +1,28 @@
+org.gradle.caching=true
+group=io.ballerina
+version=1.0.0
+
+#dependency
+ballerinaLangVersion=2.0.0-beta.3-20210821-093200-e064dcef
+testngVersion=7.3.0
+slf4jVersion=1.7.30
+org.gradle.jvmargs=-Xmx4096M
+
+#stdlib
+stdlibIoVersion=1.0.0-20210823-191300-9875786
+stdlibRegexVersion=1.0.0-20210823-191300-7ae9211
+stdlibFileVersion=1.0.0-20210823-193500-cce0858
+stdlibOsVersion=1.0.0-20210823-191800-d801ea4
+stdlibLogVersion=2.0.0-20210823-192100-5fda2fc
+stdlibMimeVersion=2.0.0-20210823-193500-3367cf9
+stdlibCryptoVersion=2.0.0-20210823-191800-0c35957
+stdlibAuthVersion=2.0.0-20210823-194100-37125bb
+stdlibHttpVersion=2.0.0-20210823-202000-25c5c95
+stdlibTimeVersion=2.0.0-20210823-191300-a4575a0
+stdlibTaskVersion=2.0.0-20210823-192400-e6a100b
+stdlibCacheVersion=3.0.0-20210823-193600-72bc08c
+stdlibJwtVersion=2.0.0-20210823-194200-67299ef
+stdlibOAuth2Version=2.0.0-20210823-194500-ed0fafb
+stdlibUuidVersion=1.0.0-20210823-193600-4e9446f
+stdlibUrlVersion=2.0.0-20210823-191400-8d6ae57
+stdlibXmldataVersion=2.0.0-20210823-191400-0dd5ee0
```

Comment:
I think these versions are now outdated

Reply:
This is updated to the Ballerina Beta 4 stable release dependencies now. Thank you.